packages/nimble-components/src/button/specs/button-visual-states.md

```diff
@@ -0,0 +1,47 @@
+# Nimble-Button New Visual Designs Implementation
+
+## Problem Statement
+
+The Visual Design spec for Nimble buttons now includes primary button states `primary` and `primary-accent` that aren't implemented in the nimble-button component yet.
+
+## Links To Relevant Work Items and Reference Material
+
+[Nimble Issue #663: Adopt nimble-button visual designs for states primary, quiet, and loud](https://github.com/ni/nimble/issues/663)
+
+[Figma Spec](https://www.figma.com/file/PO9mFOu5BCl8aJvFchEeuN/Nimble_Components?type=design&node-id=1295-77205)
+
+## Implementation / Design
+
+There are three possible values for `appearance-variant` on a button: `default`, `primary`, and `primary-accent`, each with its own color scheme based on the UI theme.
+`primary` and `primary-accent` states of buttons are primary buttons, which can be used together up to 3 times on a page.
+
+The `primary` button should be the default choice when one button needs to be emphasized.
+The `primary-accent` button can be used in situations where a button needs to have the most prominent eye-catching approach, or when there is a lack of color.
```

Comment:
I think it would be nice to have visual examples of when we think it would be appropriate to use the `primary-accent` variant (and when it would not).

Reply:
Brandon's spec in Figma has examples for these, do you think I should make note of this at the beginning of the Implementation / Design section, or should I attach an image within the spec? I think the former would be better, so we don't have to upload an image to the nimble files.

Reply:
Hmmm...I'm struggling with this. Ultimately, I'd like nimble users to be able to quickly access the UX guidance for when to use certain style variants. This HLD isn't necessarily the right place for that, and instead should likely be surfaced in the storybook of the button. Given that, I think I'm okay with it as-is.